setup.cfg

```diff
@@ -52,14 +52,13 @@ install_requires =
     click
     jsonasobj==1.3.1
     jsonasobj2>=1.0.4
-    linkml-runtime==1.1.10
+    linkml-runtime>=1.1.12
     networkx
     numpy
     pandas
     pandasql
     pyyaml
     rdflib==5.0.0
```

Comment:
Is upgrading to RDFlib 6 coming soon?

Reply:
Hasn't been on the cards for now. I could look into it if it's an easy transition.